src/sentry/models/tagvalue.py

```diff
@@ -44,6 +44,10 @@ class Meta:
 
     __repr__ = sane_repr('project_id', 'key', 'value')
 
+    @classmethod
+    def is_valid_value(self, value):
```

Comment:
`self` is _technically_ wrong in this context, but I see we do this elsewhere erroneously. :) So it's fine. I'll fix all of them in a followup.